test/tools/yulInterpreter/Interpreter.h

```diff
@@ -102,6 +102,12 @@ struct InterpreterState
 class Interpreter: public ASTWalker
 {
 public:
+	struct Scope
+	{
+		std::map<YulString, FunctionDefinition const*> visibleFunctions;
```

Comment:
Here, it should not be `visibleFunctions`, because also the functions in higher scopes are visible.